exercises/chapter8/src/Data/AddressBook/Validation.purs

```diff
@@ -1,14 +1,7 @@
 module Data.AddressBook.Validation where
 
 import Prelude
-import Data.AddressBook
-  ( Address
-  , Person
-  , PhoneNumber
-  , address
-  , person
-  , phoneNumber
-  )
```

Comment:
I like the clarity of this style, but auto-import and auto-format want the one-liner.